### PR TITLE
Features/add profile to Topnav

### PR DESCRIPTION
**Pull request by aneyzberg**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/react-hig/src/elements/components/GlobalNav/Container.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    return {higNav, higContainer};
+  }
+
+  // Create the GlobalNav context for the TopNav to be attached to
+  let Context = props => <GlobalNav > { props.children }</GlobalNav >;
+
+  it('renders a Container', () => {
+    const { higNav, higContainer } = createHigGlobalNav();
+
+    //console.log(higNav);
```
Comment (nfiniteset) [inline packages/react-hig/src/elements/components/GlobalNav/Container.test.js]: 🔪

File: packages/react-hig/src/elements/components/GlobalNav/Profile.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import HIGElement from '../../HIGElement';
+import HIGChildValidator from '../../HIGChildValidator';
+import createComponent from '../../../adapters/createComponent';
+
+
+export class Profile extends HIGElement{
+  
+  componentDidMount(){
+
+  }
```
Comment (nfiniteset) [inline packages/react-hig/src/elements/components/GlobalNav/Profile.js]: Let's lose this.

File: packages/react-hig/src/elements/components/GlobalNav/Profile.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        this.hig[mapping[propKey]](propValue);
+      } else {
+        this.commitPropChange(propKey, propValue);
+      }
+    }
+  }
+}
+
+const ProfileComponent = createComponent(Profile);
+
```
Comment (nfiniteset) [inline packages/react-hig/src/elements/components/GlobalNav/Profile.js]: Extra line here

File: packages/react-hig/src/elements/components/GlobalNav/Profile.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import * as HIG from 'hig.web';
+import * as PropTypes from 'prop-types';
+
+import HIGElement from '../../HIGElement';
+import HIGChildValidator from '../../HIGChildValidator';
```
Comment (nfiniteset) [inline packages/react-hig/src/elements/components/GlobalNav/Profile.js]: Remove if unused

Comment (nfiniteset) [inline packages/react-hig/src/elements/components/GlobalNav/Profile.js]: Remove if unused

File: packages/react-hig/src/elements/components/GlobalNav/TopNav.js [hunk 5]
```diff
@@ -50,14 +63,56 @@ export class TopNav extends HIGElement {
...
 
 TopNavComponent.propTypes = {
   logo: PropTypes.string,
   logoLink: PropTypes.string,
-  onHamburgerClick: PropTypes.func
+  onHamburgerClick: PropTypes.func,
+  addProfile: PropTypes.func,
+};
+
+TopNavComponent.propTypes = {
```
Comment (nfiniteset) [inline packages/react-hig/src/elements/components/GlobalNav/TopNav.js]: Hmm. Merge with the block above?

File: packages/react-hig/src/elements/components/GlobalNav/Profile.js (hunk 3 above)
Comment (aneyzberg) [inline packages/react-hig/src/elements/components/GlobalNav/Profile.js]: fixed

**Pull request merged**